qa-portal-angular/projects/portal-core/src/app/_common/services/qa-help.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { HelpModel } from '../models/help-model';
+import { filter, map } from 'rxjs/operators';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class QaHelpService {
+
+  private messageSource = new BehaviorSubject("");
+  currentMessage = this.messageSource.asObservable();
+  private helpContent;
+  private userType;
+
+  constructor(private router: Router, private http: HttpClient) { }
+
+  getUserType(roleBool){
+    if (roleBool == true){
+      this.userType = "USER";
+    } else {
+      this.userType = "ADMIN";
+    }
+    return this.userType;
+  }
+
+
+  accessFile(): Observable <HelpModel>{
+    return this.http.get<HelpModel>('../assets/Textfiles/homeHelp.json', {responseType: 'json'});
+  }
+
+changeMessage(data, tag){
+  console.log(tag);
+  data.subscribe(data => this.helpContent = data.filter(help => help.Tag == tag || help.Tag + String(tag).match(/[0-9]+/) == tag));
+  data.subscribe(() => this.helpContent = this.helpContent[0].Help.split("\n"));
+  data.subscribe(() => this.helpContent = this.helpContent.join("\n"));
+  data.subscribe(() => this.messageSource.next(this.helpContent));
+  };
+
+  tooltipMaker(data){
+    if (data == "Training"){
```

Comment:
Should these be hard-coded? This could be optimised by making a map of the data.

Reply:
These were hard coded because whenever I tried to map and subscribe using a json file, it would continuously loop and overload the RAM to the point that the computer becomes unusable.

Reply:
That doesn't sound right. In the interests of time can you make a new issue to move these to an external config and I'll approve this.

Reply:
Issue for config file create at issue #190